### ISSUE TITLE
[TF 2.0] Custom objects

### ISSUE THREAD

**Issue by tgpsantos** [tensorflow/tensorflow#33387]
Hello all,

I have a keras file (.hdf5) that I've saved using TF 1.14 with custom operations (eg loss function). 
I recently upgraded to Tensorflow 2.0 and I was trying to convert this model to .tflite but I've seen that there is no 'custom_objects' argument. 
Is this an unnecessary argument had I saved the .hdf5 using the latest TF 2.0? Or is there any other new way to include this custom object when converting to .tflite?

Thank you in advance

**Comment by oanush** [comment]
@tgpsantos ,
Can you please refer the [link](https://github.com/tensorflow/tensorflow/issues/20878) of similar issue ?Thanks!

**Comment by tgpsantos** [comment]
Thank you for the answer @oanush . The whole link seems like a mix of different issues but I assume you're referring to the answer that advices converting with an itermediate step, ie .h5 to .pb. Is this going to be in the right format to use in convert_from_saved_model() in TF2.0? I believe there is no convert_from_saved_graph in the latest version

PS: The original post was referring to TF1.9, I can make it work with TF1.14. My problem is with TF2.0

**Comment by jvishnuvardhan** [comment]
@tgpsantos This is a stale issue. Is this still an issue for you?

If yes, can you please share a simple standalone code to reproduce the issue. Since the issue opened, there is a lot of TFLite tutorials added to convert keras model/saved model into different levels of precisions. Please check the [guide here](https://www.tensorflow.org/lite/convert). Thanks!
